testproxy/services/create-client.js

```diff
@@ -31,7 +31,9 @@ const createClient = ({clientMap}) =>
   normalizeCallback(async rawRequest => {
     // TODO: Handle refresh periods
     const {request} = rawRequest;
-    const clientConfig = require('../../src/v2/bigtable_client_config.json');
+    const clientConfig = JSON.parse(
+      JSON.stringify(require('../../src/v2/bigtable_client_config.json'))
+    );
```

Comment:
To prevent conformance tests from modifying behaviour in other conformance tests, we must use a copy of the clientConfig so that we don't modify the original and set timeouts for tests that shouldn't be using timeouts.